desktop-widgets/divelistview.cpp

```diff
@@ -541,7 +541,7 @@ void DiveListView::reloadHeaderActions()
 			a->setChecked(shown);
 			a->setProperty("index", i);
 			a->setProperty("settingName", settingName);
-			connect(a, SIGNAL(triggered(bool)), this, SLOT(toggleColumnVisibilityByIndex()));
+			connect(a, &QAction::triggered, this, [this](int) { toggleColumnVisibilityByIndex(); });
```

Comment:
again, no clue what this does

Reply:
NB: In this case I also don't understand the point of the lambda. This should work without, no?

Reply:
Or is it the default parameter to the `triggered` signal that makes overload resolution fail?

Reply:
This goes wrong if the signal has no parameters, and the slot has a lot, but all have default parameters. You sadly can't just connect them in that case, so I took the "simple" solution of adding an intermediate step that just forwards it. But I applied that pattern a bit too often.